keymaps/project-viewer.json

```diff
@@ -1,7 +1,16 @@
 {
-  ".platform-darwin, .platform-win32, .platform-linux": {
```

Comment:
There's no `cmd` key on most Windows or Linux keyboards and no `ctrl` key on Mac keyboards, so any `cmd-ctrl` keybinding is unusable.

#46 discussed `ctrl-alt-t`, and I mostly brought it up because `tool-bar` is extremely common. I have nothing on `ctrl-alt-n` or `ctrl-alt-v`.